xml/installation-networking-l2gw5930.xml

```diff
@@ -197,18 +197,18 @@ ip address 10.1.1.10 255.255.255.0</screen>
 ip address 10.10.83.3 255.255.255.0</screen>
     <para>
      The 5930 switch has a fortygigE1/0/5 interface to which a splitter cable
-     is connected that splits the network into four tengigEthernet
+     is connected that splits the network into four tengig Ethernet
```

Comment:
I don't think splitting that word makes sense, given that it used as the interface name here, apparently. Rather put it in a literal element.